src/components/Button/src/Button.vue

```diff
@@ -104,6 +97,33 @@ function setColorVariables(tokens, variant) {
 	};
 }
 
+const TEXT_STYLES = new Set([
+	'fontFamily',
+	'fontWeight',
+	'fontSize',
+	'fontStyle',
+	'textTransform',
+	'textDecoration',
+]);
+
+function kebabCase(string) {
+	return string.replace(/([\da-z])([A-Z])/g, '$1-$2').toLowerCase();
+}
+
+function formatCssStyles(theme, styles, suffix) {
+	const formattedStyles = {};
+	for (const [style, value] of Object.entries(styles)) {
+		if (TEXT_STYLES.has(style) && cssValidator(style, value)) {
+			if (value.startsWith('@')) {
```

Comment:
i don't think you have to do this `if` check, this line:
````js
formattedStyles[`--${kebabCase(style)}${suffix}`] = theme.resolve(value);
````
works regardless of whether `value` is a pointer or not